src/modules/views.js

```diff
@@ -27,7 +27,13 @@ const Views = class {
 
   static displayTodos() {
     const todos = Persist.getLSContent();
-    todos.forEach((todo) => Views.addTodo(todo));
+    todos.forEach((todo) => {
+      if (todo.completed === true) {
+        // todo.description.classList.add('line-thru');
+      }
+      // console.log(description.value);
```

Comment:
- [ ] Please remove console.log from the codebase as mentioned in the javascript best practices we _**"Do not commit console.log to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website."**_